gtubt/lib/ui/pages/profile_page.dart

```diff
@@ -294,7 +296,13 @@ class ProfilePage extends StatelessWidget {
       builder: (context, state) {
         return BlocBuilder<UserBloc, UserState>(
           builder: (context, state) {
-            return buildAll(context, state);
+            if (state.isFailure) {
+              return Center(
```

Comment:
burada yapilmamali. BlocBuilder yerine BlocConsumer veya BlocListener ve BlocBuilder birlikte kullanilabilir. Hata mesahlerini ekrande gostermek yerine toaster message gosteriyoruz. O yuzden burada sayfanin kendisini degistirmemeliyiz.

Reply:
state.isFailure durumunda user gelmemiş demektir.Elimizde user bilgisi olmadan buildAll fonksiyonuna User bilgilerini yollamak mantıklı mı ?

Reply:
yapabilirsek bir onceki state de UI'da gosterilenleri koruyalim.

Reply:
nasıl yapcaz onu

Reply:
o is sende :D bakacan oluyor mu olmuyor mu :D